NodeBase/start-selenium-node.sh

```diff
@@ -7,6 +7,16 @@ if [ ! -e /opt/selenium/config.json ]; then
   exit 1
 fi
 
+# Start the pulseaudio server
+pulseaudio -D --exit-idle-time=-1
```

Comment:
Shouldn't this be done only for the Firefox node?

Reply:
According to my answer above https://github.com/SeleniumHQ/docker-selenium/pull/994#discussion_r380614139, `pulseaudio` daemon is required here because all browsers are affected by the memory leak caused if there is no sound consumption.

Reply:
Thanks for clarifying!